File: php/URI/1131.php
<?php

$grenais = 0;
$inter = 0;
$gremio = 0;
$empates = 0;


while (True) {
    $linha = explode(' ', readline());
    $i = $linha[0];
    $g = $linha[1];
    $grenais += 1;
    if ($i > $g)
        $inter += 1;
    else if ($i < $g)
        $gremio += 1;
    else
        $empates += 1;
    echo "Novo grenal (1-sim 2-nao)\n";
    $linha = explode(' ', readline());
    $num = $linha[0];
    if ($num == 2)
        break;
}

echo $grenais . " grenais\n";
echo "Inter:" . $inter . "\n";
echo "Gremio:" . $gremio . "\n";
echo "Empates:" . $empates . "\n";

if ($inter > $gremio)
    echo "Inter venceu mais\n";
else if ($inter < $gremio)
    echo "Gremio venceu mais\n";
else
    echo "Nao houve vencedor\n";

File: C#/URI/1131.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int flag, x, y, inter_cont = 0, gremio_cont = 0, empate = 0, grenal_cont=0;

            flag = 1;
            while (flag == 1)
            {
                string[] linha = Console.ReadLine().Split(' ');

                x = int.Parse(linha[0]);
                y = int.Parse(linha[1]);

                if (x > y)
                {
                    inter_cont++;
                }
                else if (x < y)
                {
                    gremio_cont++;
                }
                else
                {
                    empate++;
                }
                grenal_cont++;
                Console.WriteLine("Novo grenal (1-sim 2-nao)");
                flag = int.Parse(Console.ReadLine());

            }

            Console.WriteLine("{0} grenais", grenal_cont);
            Console.WriteLine("Inter:{0}", inter_cont);
            Console.WriteLine("Gremio:{0}", gremio_cont);
            Console.WriteLine("Empates:{0}", empate);

            if(inter_cont > gremio_cont)
            {
                Console.WriteLine("Inter venceu mais");
            }
            else if (inter_cont < gremio_cont)
            {
                Console.WriteLine("Gremio venceu mais");
            }
            else
            {
                Console.WriteLine("Nao houve vencedor");
            }


    }

}